tests/unit/src/main/scala/tests/TestHighlights.scala

```diff
@@ -0,0 +1,17 @@
+package tests
+
+import org.eclipse.lsp4j.DocumentHighlight
+
+object TestHighlights extends TestHighlights
```

Comment:
Can we inline this into `TestingServer` or move it into `RangeReplace`? The reason we have `TestHovers` is because we share a few utilities with the `cross` tests, which is not the case with `documentHighlight`.

Reply:
I put them in a different projects since I changed another class to use RangeReplace, which is in a separate project entirely. Moving TestHighlights doesn't seem necessary.